packages/react-native-reanimated/apple/worklets/apple/WorkletsModule.mm

```diff
@@ -52,4 +52,13 @@ @implementation WorkletsModule {
   return @YES;
 }
 
+- (void)invalidate
+{
+  // We have to destroy extra runtimes when invalidate is called. If we clean
+  // it up later instead there's a chance the runtime will retain references
+  // to invalidated memory and will crash on destruction.
+  workletsModuleProxy_.reset();
+  [super invalidate];
```

Comment:
Let's add a newline here since `[super invalidate];` is unrelated to the comment above.
```suggestion
  // We have to destroy extra runtimes when invalidate is called. If we clean
  // it up later instead there's a chance the runtime will retain references
  // to invalidated memory and will crash on destruction.
  workletsModuleProxy_.reset();

  [super invalidate];
```